src/class-wpml-cornerstone-utils.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace WPML\PB\Cornerstone;
+
+class Utils {
+
+	const MODULE_TYPE_PREFIX = 'classic:';
+
+	/**
+	 * @param array $data
+	 * @return string
+	 */
+	public static function get_node_id( $data ) {
```

Comment:
@dgwatkins, we have an informal convention to use the camelCase format when we create new code with namespaces. Could you update your code accordingly?

Reply:
Updated, thanks